scripts/west-commands.yml

```diff
@@ -34,3 +34,8 @@ west-commands:
       - name: ncs-genboard
         class: NcsGenboard
         help: generate board skeleton files for any Nordic SoC
+  - file: scripts/west_commands/provision-54l.py
```

Comment:
Should it be `script/west_commands/ncs-provision.py`? Also better it no to use `-` in python module name - it won't be importable (e.g. in unit tests).